test/bigtest/interactors/item-view-page.js

```diff
@@ -26,6 +27,10 @@ import MultiColumnListInteractor from '@folio/stripes-components/lib/MultiColumn
   clickDelete = clickable('[data-test-inventory-delete-item-action]');
 }
 
+@interactor class CirculationHistory {
+  kvValues = collection('[data-test-kv-value]');
```

Comment:
It seems, kvValues is not a clear name. I know it's used in a lot of places, but maybe we can try give more valuable name.

@VictorSoroka1  what do you think about it?